maylcf/pluralsight-courses
# [003] [Client] Web client initial setup

**Setup Instructions:**

- Navigate to the folder where your bike server is (not inside the bike folder).
- Run the following command to create your Angular application:
`ng new bike-ui -routing`

- go inside bike-ui folder and run the following project to start the application:
`npm start`

- Node Server code:
https://github.com/dlbunker/ps-spring-boot-and-angular/tree/master/node

**Testing:**

- Make sure bike server is running.
- Run client application (`npm start`)
- On the browser, try to load `http://localhost:4200/server/api/v1/bikes`
